docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
 # Math
 
-table with math blocks -> equivalent
+When transitioning into JavaScript, we see that doing calculations and writing math formulas becomes a lot easier.
 
-| blocks | JavaScript |
-| -- | -- |
-| x  |  * |
-| +  |  + | 
-| %  |  / | 
+## Math Operators
 
-etc
+When writing code with blocks, we saw that we needed to have a block for each math operator (e.g. +, -, &times;, &divide;). This can be pretty annoying and make our code formulas more complicated than they need to be. We'll see that in JavaScript, formulas are a lot more condensed.
+
+Here is how we translate each operator from blocks to JavaScript
+
+|Operation 					|Block					|Javascript			|
+|:---						|:---:					|:---				|
+|Addition (+)				|`[let x = (6 + 2)]`	|`let x = (6 + 2);`	|
+|Subtraction (-)			|`[let x = (6 - 2)]`	|`let x = (6 - 2);`	|
+|Multiplication (&times;)	|`[let x = (6 * 2)]`	|`let x = (6 * 2);`	|
+|Division (&divide;)		|`[let x = (6 / 2)]`	|`let x = (6 / 2);`	|
 
 ### Complex Expressions
 
-```typescript
-let num: number = 5 + 3 * 2;
-console.log("" + num);
+As mentioned above, when using blocks, we needed to use a separate block for each Math operator. In JavaScript, it is a lot easier to string these operators together.
+
+For example, what was 
+
+```blocks
+let num = 0
+num = 1 + (2 + 3)
 ```
 
+is now
+
 ```typescript
-let num: number = 15 / 3 * 8;
+let num: number = 1 + 2 + 3
 console.log("" + num);
 ```
 
-## Assignment Operators
-
-### Show increment / change by transition (blocks -> JavaScript)
+Some other examples are:
```

Comment:
Maybe give each of these a sub example? e.g. example #1a: addition, example #1b: multiplication, etc